src/command/mtr-command.ts

```diff
@@ -32,24 +32,30 @@ const mtrOptionsSchema = Joi.object<MtrOptions>({
 
 export const getResultInitState = () => ({hops: [], rawOutput: '', data: []});
 
-export const mtrCmd = (options: MtrOptions): ExecaChildProcess => {
-	const protocolArg = options.protocol === 'icmp' ? null : options.protocol;
+export const argBuilder = (options: MtrOptions): string[] => {
+	const intervalArg = ['--interval', process.env['NODE_ENV'] === 'development' ? '1' : '0.5'];
```

Comment:
Not directly related but this should probably go to config instead of reading ENV right here.